templates/kubernetes/docs/index.md

```diff
@@ -4,10 +4,11 @@ markdown_includes:
   nav: "shared/_side-navigation.md"
 context:
   title: "Kubernetes documentation"
-  description: Documentation for the Charmed Distribution of Kubernetes.
+  description: Documentation for Charmed Kubernetes.
+toc: False
 ---
 
-The Charmed Distribution of Kubernetes<sup>&reg;</sup> (CDK), is pure Kubernetes tested across the widest range of clouds with modern metrics and monitoring, brought to you by the people who deliver Ubuntu.
+ Charmed Kubernetes<sup>&reg;</sup>, is pure Kubernetes tested across the widest range of clouds with modern metrics and monitoring, brought to you by the people who deliver Ubuntu.
```

Comment:
It was in there previously, but the comma after "Charmed Kubernetes<sup>&reg;</sup>" doesn't need to be there.